README.md

```diff
@@ -39,6 +39,18 @@ In the training phase, the best case is that we are able to detect between Y+D a
 
 The labeling technical details are described [in this issue](https://github.com/marco-c/autowebcompat/issues/2).
 
+### Bounded-Box Labeling 
+
+This kind of labeling allows us to store the areas where the incompatibilities lie alongside the labels.
+<img src="https://user-images.githubusercontent.com/19645459/37787732-b21122b4-2e25-11e8-8d01-8d192023463f.png" />
+
+```
+Press 'y' to mark compatible.
```

Comment:
Let's put all commands in list entries:
- Press **y** to mark the images as compatible;
- ...